tmt/utils.py

```diff
@@ -852,34 +852,42 @@ def format(
     return output
 
 
-def create_directory(path, name, quiet=False):
+def create_directory(path, name, dry=False, quiet=False):
     """ Create a new directory, handle errors """
     say = log.debug if quiet else echo
     if os.path.isdir(path):
         say("Directory '{}' already exists.".format(path))
         return
     try:
-        os.makedirs(path, exist_ok=True)
-        say("Directory '{}' created.".format(path))
+        if dry:
+            say("Directory '{}' would be created.".format(path))
+        else:
+            os.makedirs(path, exist_ok=True)
+            say("Directory '{}' created.".format(path))
     except OSError as error:
         raise FileError("Failed to create {} '{}' ({})".format(
             name, path, error))
 
 
-def create_file(path, content, name, force=False, mode=0o664, quiet=False):
+def create_file(path, content, name, dry=False, force=False, mode=0o664, quiet=False):
     """ Create a new file, handle errors """
     say = log.debug if quiet else echo
     action = 'created'
```

Comment:
```suggestion
    action = 'would be created' if dry else 'created'
```
What about something like this? If we adjust the wording here (and similarly for `overwritten`) the rest of the code would be a little bit more clear.